--- pngquant-2.6.0/Makefile.orig	2016-02-21 01:19:50.000000000 +0100
+++ pngquant-2.6.0/Makefile	2016-03-06 13:17:13.470363122 +0100
@@ -43,7 +43,7 @@
 rwpng_cocoa.o: rwpng_cocoa.m
 	$(CC) -Wno-enum-conversion -c $(CFLAGS) -o $@ $< &> /dev/null || clang -Wno-enum-conversion -c -O3 $(CFLAGS) -o $@ $<
 
-$(BIN): $(OBJS) $(STATICLIB)
+$(BIN): $(OBJS) $(SHAREDLIB)
 	$(CC) $^ $(CFLAGS) $(LDFLAGS) -o $@
 
 $(TESTBIN): test/test.o $(STATICLIB)
